pontoon/base/static/js/fluent_interface.js

```diff
@@ -348,7 +348,7 @@ var Pontoon = (function (my) {
 $(function() {
 
   // Ignore editing for anonymous users
-  if (!Pontoon.user.id) {
+  if (Pontoon && Pontoon.user && !Pontoon.user.id) {
```

Comment:
The `Pontoon` object is only available in translate view. Now that we run this code elsewhere, we need an additional check.